ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/propertytest/suppliers/blocks/versions/bellatrix/SignedBlindedBeaconBlockSupplier.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.propertytest.suppliers.blocks.versions.bellatrix;
+
+import tech.pegasys.teku.spec.SpecMilestone;
+import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
+import tech.pegasys.teku.spec.propertytest.suppliers.DataStructureUtilSupplier;
+import tech.pegasys.teku.spec.util.DataStructureUtil;
+
+public class SignedBlindedBeaconBlockSupplier extends DataStructureUtilSupplier<SignedBeaconBlock> {
+
+  public SignedBlindedBeaconBlockSupplier() {
+    super(DataStructureUtil::randomSignedBlindedBeaconBlock, SpecMilestone.BELLATRIX);
```

Comment:
I think it would be valuable to make all suppliers with different milestones in this PR

Reply:
The second parameter is the minimum milestone. In this case, blinded beacon blocks are only applicable from Bellatrix. This supplier would generate random values for Bellatrix,Capella, Deneb and other future forks.